moto/cloudwatch/models.py

```diff
@@ -77,8 +77,10 @@ def __init__(
         insufficient_data_actions,
         unit,
         actions_enabled,
+        region="us-east-1",
     ):
         self.name = name
+        self.alarm_arn = make_arn_for_alarm(region, DEFAULT_ACCOUNT_ID, name)
```

Comment:
Minor nit: why not just call this property `arn`?  `FakeAlarm.alarm_arn` seems redundant, especially considering the other properties (e.g. `name` instead of `alarm_name`).